seg_lapa/config/trainer/trainer.yaml

```
@@ -0,0 +1,11 @@
+# @package _group_
+name: trainer
+gpus: "0"
+overfit_batches: 0.0
+distributed_backend: "ddp"
+num_nodes: 1
+precision: 32
+limit_train_batches: 1.0
+limit_val_batches: 1.0
+limit_test_batches: 1.0
+fast_dev_run: True
```

```suggestion
fast_dev_run: True
max_epochs: 1000
resume_from_checkpoint: null
```

I think these 2 parameters will be needed by us. Let's also add their corresponding values in the config parsing code.